conduit/fair/data/datasets/acs.py

```diff
@@ -100,14 +100,28 @@ def _employment_filter(data: pd.DataFrame) -> pd.DataFrame:
     return df
 
 
+def _adult_filter(data: pd.DataFrame) -> pd.DataFrame:
+    """Mimic the filters in place for Adult data.
+
+    Adult documentation notes: Extraction was done by Barry Becker from
+    the 1994 Census database. A set of reasonably clean records was extracted
+    using the following conditions: ((AAGE>16) && (AGI>100) && (AFNLWGT>1)&& (HRSWK>0))
+    """
+    df = data
+    df = df[df['AGEP'] > 16]
```

Comment:
It would be much more efficient to assemble a single mask and index the dataframe once with it; indexing a pandas dataframe is not cheap!